src/health_misinfo_shared/fine_tuning.py

```diff
@@ -2,7 +2,7 @@
 # Though had a few issues with it..ascii
 
 from __future__ import annotations
-from typing import Any, Iterable, Optional
+from typing import Any, Iterable, Optional, Dict
```

Comment:
this is incredibly anal, so feel free to ignore it, but somebody pointed out to me a while ago that you don't really need the upper case `Dict` anymore cos `dict` now works for type hinting. Also, doing e.g. `int | None` is preferable to `Optional` (for reasons I still don't fully understand)